### PR TITLE
upgrade navstar and minuteman

### PR DESCRIPTION
**Pull request by GoelDeepak**
## High Level Description

Updates navstar and minuteman which has a fix wherein lashup was timing out on start of marathon-lb 

## Related Issues

  - [DCOS-13430](https://dcosjira.atlassian.net/browse/DCOS-13430) Navstar crash after installing marathon-lb (DCOS 1.8.7).

## Checklist for all PR's

  - [x] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [x] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]
___
**PLEASE FILL IN THE TEMPLATE ABOVE** / **DO NOT REMOVE ANY SECTIONS ABOVE THIS LINE**

## Instructions and Review Process

**What is the review process and when will my changes land?**

All PRs require 2 approvals using github's [pull request reviews](https://help.github.com/articles/about-pull-request-reviews/).

Reviewers should be:
* Developers who understand the code being modified.
* Developers responsible for code that interacts with or depends on the code being modified.

It is best to proactively ask for 2 reviews by @mentioning the candidate reviewers in the PR comments area. The responsibility is on the developer submitting the PR to follow-up with reviewers and make sure a PR is reviewed in a timely manner. Once a PR has **2 ship-it's**, **no red reviews**, and **all tests are green** it will be included in the [next train](https://github.com/dcos/dcos/blob/master/contributing.md).

### CODE REVIEW

File: packages/minuteman/buildinfo.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
     "minuteman": {
       "kind": "git",
       "git": "https://github.com/dcos/minuteman.git",
-      "ref": "54b665a30deb8ee7c3230140273e580d98826ad7",
-      "ref_origin": "master"
+      "ref": "64952658a77a75db5f280c4c5f064012d8d323f9",
+      "ref_origin": "1.8"
```
Comment (alberts) [inline packages/minuteman/buildinfo.json]: @GoelDeepak let's call this branch dcos/1.8 for consistency. I'll delete the 1.8 branch once you update this.

**Comment by aeyakovenko** [comment]
can you post the diff of whats changed?  (1st checklist)

**Comment by GoelDeepak** [comment]
@aeyakovenko  https://github.com/dcos/lashup/pull/21

**Comment by drewkerrigan** [comment]
@aeyakovenko @GoelDeepak another sanity check view of the changes:

Minuteman Changes: https://github.com/dcos/minuteman/compare/DCOS-13228-fix-ipvs...64952658a77a75db5f280c4c5f064012d8d323f9

Navstar Changes: https://github.com/dcos/navstar/compare/19d25bdd7c7c0f794c7417ecd664a896dbaad275...30a31d23ee28d6cc49e579bcdd364d74b2f53cf2

Lashup Changes: https://github.com/dcos/lashup/compare/c10a03c4f1d2aada54b00c95ef487378a3734051...f2a20fa1033724cf47724e6c9d701f4dcd4b4e01

**Pull request merged**